### PR TITLE
wavetable morph lag adjustment

### PR DESCRIPTION
**Pull request by blancoberg**
Wavetable morph lag for modulations has been removed. 

Morph lag has been removed for all modulations as it is not needed with the improved interpolation lerp. 
The lag is still there for handling the ui interactions and daw automations. ( unmodulated value of the parameter ) 

Related #7696

### CODE REVIEW

**Comment by baconpaul** [comment]
I’m sorry Joel I didn’t get to this today

**Comment by blancoberg** [comment]
> I’m sorry Joel I didn’t get to this today

No stress at all Paul!

File: src/common/dsp/SurgeVoice.h [hunk 1]
```diff
@@ -61,6 +62,7 @@ class alignas(16) SurgeVoice
     void legato(int key, int velocity, char detune);
     void switch_toggled();
     void freeAllocatedElements();
+    void copyParametersToLocal();
```
Comment (baconpaul) [inline src/common/dsp/SurgeVoice.h:65]: It all looks really great except for this one added function which I can't find either used or defined. Code booger?

**Review by baconpaul — COMMENTED** [review]
Really lovely work Joel

One comment I couldn't understand but the rest of it looks correct to me.

File: src/common/dsp/SurgeVoice.h (hunk 1 above)
Comment (blancoberg) [reply src/common/dsp/SurgeVoice.h:65]: > It all looks really great except for this one added function which I can't find either used or defined. Code booger?

Yeah, I missed that one. I used it in a previous attempt where tried to copy the localdata at the voice level. I then realised that it wouldn't work as the scene modulations would already have been copied over. 

I'll remove it

Comment (baconpaul) [reply src/common/dsp/SurgeVoice.h]: Thanks!

**Pull request merged**